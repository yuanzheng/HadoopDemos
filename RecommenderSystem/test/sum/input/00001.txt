1346432:79	0.009676709918627667
1346432:79	0.2004984971776263
1346432:79	0.3770984531925812
305344:79	0.0019060186203357526
305344:79	0.002419177479656917
305344:79	0.002492485888131369
305344:79	0.0011729345355912322
305344:80	0.003477396920019871
305344:80	0.037754595131644315


