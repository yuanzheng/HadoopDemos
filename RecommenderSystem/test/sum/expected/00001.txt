1346432:79	0.5872736602888351
305344:79	0.00799061652371527
305344:80	0.04123199205166419